src/scriptworker/constants.py

```diff
@@ -453,11 +470,12 @@
                     ),
                     "mobile": immutabledict(
                         {
+                            "android-components-repo": ("/mozilla-mobile/android-components",),
                             "fenix-repo": ("/mozilla-mobile/fenix",),
+                            "fennec-profile-manager": ("/mozilla-mobile/fennec-profile-manager",),
```

Comment:
question: did we want a trailing `-repo` to be consistent with others or should we not bother given that this is going away once adhoc signing is up?

Reply:
Oops, you're right! Fixed!